sci-geosciences/gmt/gmt-4.5.17.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+AUTOTOOLS_AUTORECONF=yes
+
+inherit autotools-utils multilib
+
+GSHHS="gshhs-2.2.0"
+
+DESCRIPTION="Powerful map generator"
+HOMEPAGE="http://gmt.soest.hawaii.edu/"
+SRC_URI="
+	mirror://gmt/${P}-src.tar.bz2
+	mirror://gmt/legacy/${GSHHS}.tar.bz2
+	gmttria? ( mirror://gmt/${P}-non-gpl-src.tar.bz2 )"
+
+LICENSE="GPL-2 gmttria? ( Artistic )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug +gdal gmttria +metric mex +netcdf octave postscript"
+
+RDEPEND="
+	!sci-biology/probcons
+	gdal? ( sci-libs/gdal )
+	netcdf? ( >=sci-libs/netcdf-4.1 )
+	octave? ( sci-mathematics/octave )"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/GMT${PV}"
+
+# mex can use matlab too which i can't test
+REQUIRED_USE="
+	mex? ( octave )
+"
+
+# hand written make files that are not parallel safe
+MAKEOPTS+=" -j1"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-4.5.9-no-strip.patch
+	"${FILESDIR}"/${PN}-4.5.6-respect-ldflags.patch
+	)
+
+AUTOTOOLS_IN_SOURCE_BUILD=1
+
+src_prepare() {
+	mv -f "${WORKDIR}/share/"* "${S}/share/" || die
+
+	tc-export AR RANLIB
```

Comment:
inherit toolchain-funcs?

Reply:
Removed tc-export. This is part of the old and outdated bugfix.